_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -48,7 +48,7 @@ Using `aria-hidden="false"` on a descendant of an element with `aria-hidden="tru
 
 By adding `aria-hidden="true"` to an element, content authors ensure that assistive technologies will ignore the element. This can be used to hide parts of a web page that are [pure decoration](https://www.w3.org/TR/WCAG21/#dfn-pure-decoration), such as icon fonts - that are not meant to be read by assistive technologies.
 
-A [focusable][] element with `aria-hidden="true"` is ignored as part of the reading order, but still part of the focus order, making its state of [visible](#visible) or hidden unclear.
+An element that is part of the [sequential focus navigation][] with `aria-hidden="true"` is ignored as part of the reading order, but still part of the focus order, making its state of [visible](#visible) or hidden unclear.
```

Comment:
I'm actually not sure what this AS note meant in the first place 🤔 Notably what does visibility has to do with `aria-hidden`. Was it supposed to referred to [programatically hidden](https://act-rules.github.io/glossary/#programmatically-hidden) rather than visibility?

Seems to have been here [from the start of the rule](https://github.com/act-rules/act-rules.github.io/pull/175/files#diff-695b07cb3f307e2d42f30d9af89d1ba529ab49d749f451d4f8738edc0ee767d6R41), though… So not related to this PR at all.

Reply:
My reading of this is that it would be reachable by a screen reader user when tabbing, but they wouldn't find it when using other reading modes (item navigation). visible is probably the wrong word. "discoverable" maybe?

Reply:
Lets try to elaborate on this a bit. It can certainly be clearer.

Reply:
Part of this was already said in the prior sentence. I reworded this to tie those two together better and elaborate slightly. Let me know if that makes more sense.